google/cloud/storage/benchmarks/benchmark_parser_test.cc

```diff
@@ -43,6 +43,17 @@ TEST(StorageBenchmarksUtilsTest, ParseDuration) {
   EXPECT_EQ(s(1800).count(), ParseDuration("1800s").count());
 }
 
+TEST(StorageBenchmarksUtilsTest, ParseBoolean) {
+  EXPECT_EQ(true, ParseBoolean("", true));
+  EXPECT_EQ(false, ParseBoolean("", false));
+
+  EXPECT_EQ(true, ParseBoolean("true", false));
+  EXPECT_EQ(true, ParseBoolean("True", false));
+
+  EXPECT_EQ(false, ParseBoolean("false", true));
+  EXPECT_EQ(false, ParseBoolean("False", false));
```

Comment:
Did you mean that last `false` to be `true`?

Reply:
Yes, thanks for noticing. Fixed.